.ci/install.sh

```diff
@@ -1,12 +1,12 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
-set -ex
+set -e
+set -x
 
 if [[ "$(uname -s)" == 'Darwin' ]]; then
     brew update || brew update
     brew outdated pyenv || brew upgrade pyenv
     brew install pyenv-virtualenv
-    brew install cmake || brew upgrade cmake || true
```

Comment:
why skip cmake?

Reply:
Copy of `.ci/install.sh` of `conan-templates`:
https://github.com/bincrafters/conan-templates/blob/387bf5d7a98fe5810c0438a502d8ba528f92d5b2/.ci/install.sh#L9